chart/otomi/values.yaml

```diff
@@ -81,3 +81,7 @@ charts:
 #   adminGroupID: ''
 #   # IDP group id used to identify team admin
 #   teamAdminGroupID: ''
+
+# WARNING: setting this to ANY VALUE removes ALL cluster resources 
```

Comment:
So this is is interesting use case.
To me otomi chart is like operator. 
If I delete operator shall all resources` created by operator` be deleted?
I would say no by default. In this regard I would suggest to make `destroyUponRemoval: false'
If I do `helm uninstall` the how do I prevent from removing the platform workloads?